utils/grpc/runtime.go

```diff
@@ -18,3 +25,44 @@ func (g *RuntimeClientBuilder) WithContext(ctx context.Context, f func(ctx conte
 	client sharedpb.RuntimeServiceClient) error) error {
 	return g.doWithContext(ctx, f)
 }
+
+type RuntimeServer struct {
+	DB_URL      string
+	RD_URL      string
+	serviceName string
+	sharedpb.UnimplementedRuntimeServiceServer
+}
+
+const (
+	runtimeHandlerError = "RuntimeHandler Error!"
+)
+
+//implementation of the shared helper function
+func (s *RuntimeServer) CheckRuntime(ctx context.Context, req *sharedpb.SignedRuntimeInfoRequest) (*sharedpb.RuntimeInfoReport, error) {
+	//get connection object
+
+	connection := db.ConnectionUrls{
+		PgURL: s.DB_URL,
+		RdURL: s.RD_URL,
+	}
+
+	//check runtime in shared
+	res, err := utils.CheckRuntime(ctx, req, connection)
+	if err != nil {
+		log.Error(runtimeHandlerError, zap.Error(err))
```

Comment:
`return nil, err`

Unless you want to return error message in `res` (currently not supported...)

Reply:
okay. nil, err is perfect.